tensorflow/python/ops/ragged/ragged_reduce_op_test.py

```diff
@@ -39,6 +39,16 @@ def mean(*values):
   return 1.0 * sum(values) / len(values)
 
 
+def variance(*values):
```

Comment:
Suggestion: have this just return `np.var(values)`.  (Your implementation looks right to me, but if we use a standard/reference implementation then it's one less thing to think about.)

Reply:
done